clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -7679,8 +7679,7 @@ void OffloadBundler::ConstructJobMultipleOutputs(
     else
       TypeArg = "aoo";
```

Comment:
I have to apologize that I'm kind of stuck with understanding the change. A question that could hopefully clear up the concept: why is `.aoo` type retained and what purposes does it fit after this change? Especially if the commit message suggests that we're "moving away" from using it.

Reply:
general usage is moving away from `aoo` in favor of `a` but we will continue to use `aoo` for FPGA usage - dealing with aocx/aocr archives.